signer/src/storage/postgres.rs

```diff
@@ -1442,6 +1443,60 @@ impl super::DbRead for PgStore {
         .map_err(Error::SqlxQuery)
     }
 
+    async fn key_rotation_exists(
+        &self,
+        chain_tip: &model::BitcoinBlockHash,
+        signer_set: &BTreeSet<PublicKey>,
+        aggregate_key: &PublicKey,
+        signatures_required: u16,
+    ) -> Result<Option<model::RotateKeysTransaction>, Error> {
```

Comment:
nit: this should probably return a `bool`. Or maybe we should rename the function.

Reply:
Done in [3a726de](https://github.com/stacks-network/sbtc/pull/815/commits/3a726dea891cbc88b321658ef5c500cb93a6f36e)